src/server/tran_server.cpp

```diff
@@ -625,7 +625,7 @@ tran_server::connection_handler::is_idle ()
 tran_server::ps_connector::ps_connector (tran_server &ts)
   : m_ts { ts }
   , m_daemon { nullptr }
-  , m_terminate { false }
+  , m_terminate { true }
```

Comment:
Nit:

Leave the initialize value as false and add 'if (uses_remote_storage())' conditon before calling the tran_server::ps_connector::terminate () for code consistency. There is no rules but other kind of m_terminate_xxx member value usually set to false as initial value.

Reply:
Some of the classes do it like that if their internal thread is not started automatically when creating an object. 
See `request_queue_autosend<ReqQueue>::start_thread ()` and `request_server<MsgId>::start_thread ()`.

Reply:
@wrlawodms Do you mean m_shutdown flag? The previous comment is 'Nit'. So, it is your choice. I just leave a comment in point of code consistency about m_terminate flag and ps_connector is only created when uses_remote_storage().

Reply:
I know you left it at my choice :). I just wanted to clarify it and inform you. Anyway, on second thought, I agree with you in that the ps_connector start only when `if(uses_remote_storage())`, so it should be terminated only in the same condition. Thanks.

Reply:
We've decided to go as it is with http://jira.cubrid.org/browse/LETS-767.